MsCorePkg/Library/MemoryProtectionExceptionHandlerLib/MemoryProtectionExceptionHandlerLib.inf

```diff
@@ -49,5 +49,8 @@
 [Guids]

```

Comment:
PcdLib should be explicitly added to the [LibraryClasses] section of this inf.

Reply:
Added